streaming/java/streaming-runtime/src/main/java/io/ray/streaming/runtime/barrier/Barrier.java

```diff
@@ -0,0 +1,42 @@
+package io.ray.streaming.runtime.barrier;
+
+import com.google.common.base.MoreObjects;
+import java.io.Serializable;
+import java.util.Objects;
+
+public final class Barrier implements Serializable {
```

Comment:
Add some comment?

Reply:
this data structure is useless, I removed it.